website/static/js/signIn.js

```diff
@@ -15,19 +15,10 @@ var SignInViewModel = oop.extend(formViewModel.FormViewModel, {
     constructor: function () {
         var self = this;
         self.super.constructor.call(self);
-        var existingUserEmail = decodeURIComponent($osf.urlParams().existing_user);
```

Comment:
If we're removing this, then we don't need to pass this query param anymore:

![osf___sign_in](https://cloud.githubusercontent.com/assets/2379650/15782761/81756ef6-2979-11e6-85c6-5a45c48e343e.png)